backend/shared/models/src/migrations/1751545537-create-uitpas-tokens.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE `uitpas_tokens` (
+   `id` varchar(36) CHARACTER SET ascii COLLATE ascii_general_ci NOT NULL,
```

Comment:
We zijn afgestapt van de ASCII ids - dus deze mogen de default character set en collate gebruiken:
```suggestion
  `id` varchar(36) NOT NULL,
```